src/services/Challenge/Challenge.js

```diff
@@ -1021,6 +1021,44 @@ export const saveChallenge = function (
         ).toISOString();
       }
 
+      // Validate the fields before saving
+      const {
+        instruction,
+        description,
+        name
+      } = challengeData;
+
+      if (
+        instruction === undefined ||
+        instruction.length < 150 ||
+        instruction.split(' ').length < 20 ||
+        description === undefined ||
+        description === '' ||
+        name === undefined ||
+        name.length <= 3
```

Comment:
you can simplify this using optional chain operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining